packages/daemon/test/runner/helper.test.ts

```diff
@@ -4,6 +4,8 @@ import * as helper from '../../src/utils';
 import * as sinon from 'sinon';
 import * as fs from 'fs-extra';
 import { join } from 'path';
+import * as request from 'request-promise';
+const Bluebird = require( 'bluebird');
```

Comment:
Shall we use `Promise` instead?

Reply:
Theoretically, `Bluebird` is compatible with `Promise`, but the data types are different, so we should mark the Promise object as any.
Updated.

Reply:
I can't get that why do we mark the `Promise` object as `any`?

Reply:
`Bulebird` is not completely the same with `Promise`, it will fail when check data type by typescript.